### PR TITLE
[ACCOUNT] Split pages

### LINKED ISSUE #564: Re-arrange pages and menu

### PR DESCRIPTION
**Pull request by W3stside**
# Summary

Closes #564 

image

## Testing
1. head to account page
2. click links
3. check each page

**Note:** Wasn't able to test vCOW balance/lGNO in dev, will try in pr link

### CODE REVIEW

**Review by alfetopito — COMMENTED** [review]
1. I don't think you need the title when it's already in the side tab
2. While a page is loading, the menu is not shown (only saw this on first load). Could we load them independently?

**Comment by fairlighteth** [comment]
Looks good. Some feedback:
1. I'd keep the title as you have. You see it on the Affiliate page where the menu has 'Affiliate' and the page title is 'Affiliate Program'. Which allows for flexibility and IMO makes it clearer for the user.
2. Same observation as @alfetopito about the menu. Though the existing FAQ page has the same effect.
3. The affiliate page content misses the outer container (like on the other pages)

File: src/custom/pages/Account/styled.tsx [hunk 1]
```diff
@@ -658,3 +658,31 @@ export const CardsSpinner = styled(SpinnerLoader)`
     stroke: ${({ theme }) => (theme.darkMode ? theme.text1 : theme.primary1)};
   }
 `
+
+interface TimeProps {
+  date: string | undefined
+}
+
+export const TimeFormatted = ({ date }: TimeProps) => {
```
Comment (fairlighteth) [inline src/custom/pages/Account/styled.tsx:666]: Should we move to an imported util function?

**Comment by nenadV91** [comment]
@alfetopito I'm not sure if you are referring to when you switch the account pages how to sidebar also disappears. If so I think that the solution for this would be have one parent route for the account pages where you would render sidebar and the content, and then the content would render sub-routes for the 4 pages we have. But that is if we want to go in this direction.

**Review by nenadV91 — APPROVED** [review]
Apart from the mentioned things LGTM

File: src/custom/pages/Account/styled.tsx (hunk 1 above)
Comment (W3stside) [reply src/custom/pages/Account/styled.tsx:666]: 🤷 we could but it's only used in 1 place and is a component

**Comment by W3stside** [comment]
> 1. I don't think you need the title when it's already in the side tab
> 2. While a page is loading, the menu is not shown (only saw this on first load). Could we load them independently?

no
but really yes

**Comment by W3stside** [comment]
@alfetopito @fairlighteth done in [97d0f11](https://github.com/cowprotocol/cowswap/pull/826/commits/97d0f1193399871046916048b19066d9c181f138)

**Review by alfetopito — APPROVED** [review]

**Comment by W3stside** [comment]
merging but @fairlighteth let's review the base PR #619

**Pull request merged**